src/core/render/compiler/link.js

```diff
@@ -28,6 +28,12 @@ export const linkCompiler = ({
       }
 
       href = router.toURL(href, null, router.getCurrentPath());
+
+      if (config.target) {
+        attrs.push(
+          href.indexOf('mailto:') === 0 ? '' : `target="${linkTarget}"`
```

Comment:
Does the empty string in the `attrs` array serve a purpose?

I'm not familiar with this part of the code so perhaps this is what needs to be done, but if the empty string does not serve a purpose (i.e., it's skipped when `attrs` is processed) then perhaps we should avoid it using a simple `if` statement:

```js
if (config.target) {
  if (href.indexOf('mailto:') !== 0) {
    attrs.push(`target="${linkTarget}"`)
  }
}
```

Or a single-line conditional like this:

```js
if (config.target) {
  href.indexOf('mailto:') !== 0 && attrs.push(`target="${linkTarget}"`);
}
```

Reply:
Got it !

Reply:
Just my preference, but:

```js
if (config.target && href.indexOf('mailto:') !== 0) {
  attrs.push(`target="${linkTarget}"`);
}
```